HelpSource/Classes/Object.schelp

```diff
@@ -652,6 +662,41 @@ foo {
 }
 ::
 
+
+method::doesNotUnderstand
+Not usually called in user code.
+This is called when an object receives a message it does not understand.
+Object's implementation will throw an error, but other classes (link::Classes/IdentityDictionary::) might override this.
+
+argument::selector
+The name of the message as a link::Classes/Symbol::.
+
+argument::... args
+The arguments passed.
+
+code::
+4.someThing(2) // ERROR: Message 'someThing' not understood.
+::
+
+method::doesNotUnderstandWithKeys
+Not usually called in user code.
+This is called when an object receives a message it does not understand, but there are keyword arguments.
+Object's implementation will throw an error, but other classes (link::Classes/IdentityDictionary::) might override this.
+
+argument::selector
+The name of the message as a link::Classes/Symbol::.
+
+argument::argsArray
+The arguments passed that did not have keywords attached passed as an link::Classes/Array::
```

Comment:
suggestion:

An link::Classes/Array:: of arguments that have been passed directly and not via keywords.